src/mcts/node.h

```diff
@@ -362,6 +362,10 @@ class EdgeAndNode {
     return numerator * GetP() / (1 + GetNStarted());
   }
 
+  float GetNewU(float numerator) const {
+    return numerator * GetP() / std::pow(1.0 + GetNStarted(), 1.5);
```

Comment:
Should consider using fastpow instead of pow at some point, but maybe not yet.

Reply:
Are you referring to [this](https://martin.ankerl.com/2012/01/25/optimized-approximative-pow-in-c-and-cpp/) or is there another `fastpow`?

Reply:
the old policy_softmax_temperature implementation did fastpow(x,y) by fastexp2(fastlog2(x) * y) IIRC - using the fastlog2 and fastexp2 functions we have in our fastmath utilities file.